src/utils/handlerErrorShow.ts

```diff
@@ -0,0 +1,9 @@
+export const handlerErrorShow = (
+  error: string,
+  setErrorMessage: React.Dispatch<React.SetStateAction<string>>,
+): void => {
```

Comment:
You can move this handler into your component, so you don't have to pass the `setErrorMessage` every time